bdd/login.md

```diff
@@ -0,0 +1,23 @@
+Scenario: A user make a request to the authentication in the system.
```

Comment:
This file needs a rework since each request behaviour should be a new `Scenario`.

Reply:
Can you have a look on fcdb805b024578039080665a8dc5ed8e9b0a9f00 and tell me what you think? Thanks in advance.

Reply:
The idea is having scenarios per behaviour, example:

```
Scenario: A user make a request to the authentication in the system with bad formated credentials.
Scenario: A user make a request to the authentication in the system with empty payload.
...
```

Reply:
Resolved on bac2b7574168afe1ec6c2890edede91e8c9cafc8.